Makefile

```diff
@@ -16,10 +16,10 @@ generate-plugin-for-tests:  ## Generate a plugin using the cookiecutter defaults
 	# The cookiecutter has many example code blocks prefixed with '### '.
 	# To help ensure that the example code would work if the user uncommented it,
 	# we remove the all '### ' occurances before running tests.
-	sed -i 's/### //' tutor-contrib-myplugin/tutormyplugin/plugin.py 
+	sed -i 's/### //' tutor-contrib-myplugin/tutormyplugin/plugin.py
 	# We must also create this init task template, which the cookiecutter
 	# doesn't generate because its usage in plugin.py is commented out.
-	touch tutor-contrib-myplugin/tutormyplugin/templates/myplugin/jobs/init/lms.sh
+	touch tutor-contrib-myplugin/tutormyplugin/templates/myplugin/tasks/myplugin/init.sh
```

Comment:
@regisb , Emad has pointed out that we are not being consistent with what we call the directory which contains shell scripts for do-commands. It seems like all the official plugins call the directory `/tasks/`, but this cookiecutter as well as Tutor itself currently call it `/jobs/`.

Of course, it's arbitrary what we call the directory, since the shell scripts are just loaded up into `CLI_DO_COMMANDS`. Still, I agree with Emad that being consistent could makes life easier for people who are learning the plugin API.

I propose that we accept Emad's change here and call the directory `/tasks/`, and then file a backlog item to make that change in Tutor when we have time. My thinking is that updating Tutor core is easier than updating all the plugins.

What do you think?

Reply:
Yes, I agree. I'm not sure we'll be able to easily rename all instances of "jobs" (renaming the "docker-compose.jobs.yml" file and "*-docker-compose-jobs-services" patches will be a backward-incompatible change) but in general we should strive to use "tasks", which is the right term.

Reply:
Sounds good 👍🏻 

Here's the ticket to rename the /jobs/ dir to /tasks/ in tutor core: https://github.com/openedx/wg-developer-experience/issues/180

> I'm not sure we'll be able to easily rename all instances of "jobs" (renaming the "docker-compose.jobs.yml" file and "*-docker-compose-jobs-services" patches will be a backward-incompatible change) but in general we should strive to use "tasks", which is the right term.

Yeah, that would definitely be a major backwards-incompatible change. Moreover, I think it's fair to continue to call those files "jobs", because we had established earlier that "Tutor jobs" are collections of "Tutor tasks" which can be run with `tutor ... do`. With that terminology in mind, the patch and .yml file *are* used to run "jobs".